sql/mysql/inspect.go

```diff
@@ -260,6 +261,11 @@ func (i *inspect) addColumn(s *schema.Schema, rows *sql.Rows) error {
 		}
 		c.Attrs = append(c.Attrs, a)
 	}
+
+	if typ.String == "uuid" && !i.SupportsUUIDDataType() {
+		return errors.New("uuid datatype is not supported for the current database version")
+	}
+
```

Comment:
No need for this check. If it comes from the database, we can assume it's valid. 

```suggestion
```

Reply:
Thanks for the review, that makes sense I removed the check. As i was assuming that from the ENT project the schema is created on the fly and not coming directly from an existing database.